app/controllers/image_uploads_controller.rb

```diff
@@ -49,8 +50,8 @@ def limit_uploads
   def validate_image
     images = Array.wrap(params["image"])
     return if images.blank?
-    return IS_NOT_FILE_MESSAGE unless valid_image_files?(images)
-    return FILENAME_TOO_LONG_MESSAGE unless valid_filenames?(images)
+    return is_not_file_message unless valid_image_files?(images)
```

Comment:
Where are these two methods defined?

Reply:
Possibly forgot to stage. Let me check it later.

Reply:
The class seems to read it from concerns/image_uploads.rb, so seems working this way. I added one more file under app/services that is affected by this name change to the PR.